semgrep_output_v1.atd

```diff
@@ -1680,6 +1680,46 @@ type ci_scan_complete_stats = {
     <python repr="dict">
     <ts repr="map">
     option;
+
+  (* since 1.98.0 *)
+  (* In collaboration with the Data Science team, it was suggested
+   * that we start to group stats by product for organizational purposes.
+   *)
+  ?supply_chain_stats: supply_chain_stats option;
```

Comment:
FYI, now that our synthetic tests are using the atd-defined stuff, when we add fields we will have to add them to the synthetic tests as well. SMS team can handle this, but please continue to keep us in the loop - specifically on PR's that bump semgrep-interfaces versions within semgrep-app.

Reply:
Gotcha! Thanks for the heads up @chmccreery.